spec/commands/newgem_spec.rb

```diff
@@ -116,19 +119,20 @@ def gem_skeleton_assertions(gem_name)
     let(:gem_name) { "test_gem" }
     let(:generated_gem) { Bundler::GemHelper.new(bundled_app(gem_name).to_s) }
 
-    context "git config user.name present" do
+    context "git config github.user present" do
       before do
         execute_bundle_gem(gem_name)
       end
 
       it "contribute URL set to git username" do
         expect(bundled_app("test_gem/README.md").read).not_to include("[USERNAME]")
+        expect(bundled_app("test_gem/README.md").read).to include("github.com/bundleuser")
       end
     end
 
-    context "git config user.name is absent" do
+    context "git config gihhub.user is absent" do
```

Comment:
That's already accounted for 4 times.

https://github.com/danielpclark/bundler/blob/54de051a2e24bddefa85bbb1ffadadacd6800f2d/spec/commands/newgem_spec.rb#L56 ,
https://github.com/danielpclark/bundler/blob/54de051a2e24bddefa85bbb1ffadadacd6800f2d/spec/commands/newgem_spec.rb#L67 ,
https://github.com/danielpclark/bundler/blob/54de051a2e24bddefa85bbb1ffadadacd6800f2d/spec/commands/newgem_spec.rb#L253 , https://github.com/danielpclark/bundler/blob/54de051a2e24bddefa85bbb1ffadadacd6800f2d/spec/commands/newgem_spec.rb#L543